cloner.go

```diff
@@ -19,12 +22,23 @@ var (
 )
 
 func newCloner(mutate bool, capacity int) *cloner {
+	frozenConcurrency := os.Getenv("FROZEN_CONCURRENCY")
+	var maxConcurrency int
+	var err error
+	if strings.ToLower(frozenConcurrency) == "off" {
+		maxConcurrency = 1<<(bits.UintSize-1) - 1
+	} else {
+		maxConcurrency, err = strconv.Atoi(frozenConcurrency)
+		if err != nil {
+			maxConcurrency = 15
```

Comment:
Don't fail silently.

Reply:
Would adding a log be enough? I don't think the program should stop.

Reply:
Why not?

Reply:
I think that it's inconvenient if a library depends on an envvar and because of that program fails. But yes, it shouldn't fail silently, would a log be enough then?

Reply:
Errors are meant to be inconvenient.